Resources/Prototypes/Entities/Objects/Fun/EmergencyPartyKit.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  id: EmergencyPartyKit
+  parent: BaseItem
+  name: emergency party kit
+  description: For when you need a party STAT and cant wait for cargo.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/EmergencyPartyKit.rsi
+    heldPrefix: EmergencyPartyKit
```

Comment:
I'm not great with YAML but I've seen that `heldPrefix` is generally under `Item` types, whereas under `Sprite` types you want to use `state`